patches.txt

```diff
@@ -0,0 +1 @@
+bench.patches.v3.deprecate_old_config
```

Comment:
Hi Neil,

No it's not.

Regards,
cksgb

On Sep 5, 2016 09:28, "Neil Trini Lasrado" <notifications@github.com> wrote:

> In patches.txt
> <https://github.com/frappe/erpnext/pull/6223#discussion_r77485918>:
>
> > @@ -0,0 +1 @@
> > +bench.patches.v3.deprecate_old_config
>
> is this intentional ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/frappe/erpnext/pull/6223/files/3f90862ee2066ca7cfb29b2b1624f8062e292b62#r77485918>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUJjz_lNBtw4_TcjVL2zL0VANn8329wQks5qm9K6gaJpZM4JwiG5>
> .
>